src/reporter/coverage-sonar-report.xsl

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:x="http://www.jenitennison.com/xslt/xspec"
+  xmlns:test="http://www.jenitennison.com/xslt/unit-test" exclude-result-prefixes="xs" version="3.0">
+
+  <xsl:output indent="yes" exclude-result-prefixes="#all" omit-xml-declaration="yes"/>
+
+  <xsl:import href="coverage-report.xsl"/>
+
+  <xsl:template match="/">
+    <xsl:apply-templates select="." mode="test:coverage-sonar-report"/>
+  </xsl:template>
+
+  <xsl:template match="/" mode="test:coverage-sonar-report" exclude-result-prefixes="#all">
+    <xsl:variable name="context" as="node()*" select="trace/m"/>
+    
+    <coverage version="1">
+      <xsl:for-each select="$context[not(contains(@u,'xspec-utils.xsl'))]">
```

Comment:
Why does it need this special handling of `xspec-utils.xsl` whereas `coverage-report.xsl` does not need it?

Reply:
I think the goal of "sonar-coverage.xsl" is to create a project coverage report, which SonarQube will use to analyze the code.

For me, 'xspec-utils.xsl' should not be tested because:
-this style sheet is not part of the project,
-and it can lower the coverage rate.

I put a photo,
![coverage](https://user-images.githubusercontent.com/53631092/84399535-445b4b80-ac01-11ea-8499-183ab95e7334.png)
left, which I want.
right, the result if I don't exclude xspec-utils.xsl

Reply:
Yes, that's the same goal as `coverage-report.xsl`. So I'm wondering.

Reply:
#1014 removes `xspec-utils.xsl` from the coverage report XML.